app/helpers/availability_zone_helper.rb

```diff
@@ -1,3 +1,7 @@
 module AvailabilityZoneHelper
   include_concern 'TextualSummary'
+
+  def accessible_select_event_types
+    [[_('Management Events'), 'timeline']]
```

Comment:
shouldn't timeline be i18n too?

Reply:
and tests please ? :blush:

Reply:
It's value in select picker not text displayed to user so it should never be translated :)

Reply:
And rspec is added :)